mllib/src/main/scala/org/apache/spark/ml/optim/WeightedLeastSquares.scala

```diff
@@ -133,7 +133,8 @@ private[ml] class WeightedLeastSquares(
         return new WeightedLeastSquaresModel(coefficients, intercept, diagInvAtWA, Array(0D))
       } else {
         require(!(regParam > 0.0 && standardizeLabel), "The standard deviation of the label is " +
-          "zero. Model cannot be regularized with standardization=true")
+          "zero. Model cannot be regularized with standardizeLabel=true " +
+          "(standardizeLabel is not exposed in the LinearRegression API)")
```

Comment:
The second line isn't meaningful to the caller. I'd just say "Model cannot be regularized when labels are standardized".

Reply:
Good point. Since the primary way to get here is through calling LinearRegression(), I was trying to give the user a message that indicates it's a current limitation of the API, not the implementation/algorithm. I updated to your suggestion though as I do think it's an improvement over the current message of `standardization=true`, which was simply incorrect. Thanks for reviewing!